From 308a483a45d5e912a18655192e28ba629b4c1907 Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 25 Jun 2010 02:34:22 +0100
Subject: [PATCH] Revert "tpm: autoload tpm_tis based on system PnP IDs"

This reverts commit a798d8b0cb51bb966763e0feafcbff7097eb3978.

Initialisation of tpm_tis can delay boot for up to 6 minutes on
systems where the TPM doesn't respond.

See: <http://bugs.debian.org/584273>
---
 drivers/char/tpm/tpm_tis.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/char/tpm/tpm_tis.c b/drivers/char/tpm/tpm_tis.c
index 2405f17..27e8de4 100644
--- a/drivers/char/tpm/tpm_tis.c
+++ b/drivers/char/tpm/tpm_tis.c
@@ -637,7 +637,6 @@ static struct pnp_device_id tpm_pnp_tbl[] __devinitdata = {
 	{"", 0},		/* User Specified */
 	{"", 0}			/* Terminator */
 };
-MODULE_DEVICE_TABLE(pnp, tpm_pnp_tbl);
 
 static __devexit void tpm_tis_pnp_remove(struct pnp_dev *dev)
 {
-- 
1.7.1

